### PR TITLE
Meta-data changes and bug fix

### PR DESCRIPTION
**Pull request by olofsvensson**
- I moved 'MedataManagerClient' to 'ESRF/ESRFMetadataManagerClient'
- I moved the meta-data code from ID30A1MultiCollect to ESRFMultiCollect
- I also fixed a bug concerning the generation of input files for hkl2000 and mosflm. This bug is strange because it seems like some old code made it back to this branch.

These changes have been tested on MASSIF 1.

### CODE REVIEW

File: ESRF/ESRFMultiCollect.py [hunk 1]
```diff
@@ -328,11 +331,50 @@ def init(self):
...
+            beamline = "unknown"
+            proposal = "unknown"
+            if listDirectory[1] == "data":
+                if listDirectory[2] == "visitor":
+                    beamline = listDirectory[4]
+                    proposal = listDirectory[3]
+                else:
+                    beamline = listDirectory[2]
+                    proposal = listDirectory[4]
+            metadataManagerName = 'id30a1/metadata/ingest'
```
Comment (mguijarr) [inline ESRF/ESRFMultiCollect.py]: As you said in your email Olof the Tango device names have to be put in xml file

**Comment by olofsvensson** [comment]
Ok, meta data manager name and meta experiment name now moved to config file

**Comment by mguijarr** [comment]
It would be great to do the same PRs for 2.2 and master branches. I hope we will soon stop to add features in 2.1... grrr

**Comment by olofsvensson** [comment]
Ok, I'll do PRs for master and 2.2

**Pull request merged**